src/webvr-polyfill.js

```diff
@@ -95,6 +95,18 @@ WebVRPolyfill.prototype.enablePolyfill = function() {
 
   // Provide the VRDisplay object.
   window.VRDisplay = VRDisplay;
+
+  // Provide vrEnabled
+  var that = this;
+  Object.defineProperty(navigator, 'vrEnabled', {
+    get: function () {
+      return that.isCardboardCompatible() &&
+        (document.fullscreenEnabled ||
```

Comment:
oh, sorry I'm getting confused with the old API - conflating it with Fullscreen for WebVR 1.0 `<iframe allowvr>`. our naming doesn't help heh